CHANGELOG/CHANGELOG-1.15.md

```diff
@@ -0,0 +1,81 @@
+- Replace `net.IP` with `netip.Addr` in FlowExporter which optimizes the memory usage and improve the performance of the FlowExporter. ([#5532](https://github.com/antrea-io/antrea/pull/5532), [@antoninbas])
+- Update kubemark from v1.18.4 to v1.29.0 for antrea-agent-simulator. ([#5820](https://github.com/antrea-io/antrea/pull/5820), [@luolanzone])
+- Update Antrea ROADMAP.md document. ([#5807](https://github.com/antrea-io/antrea/pull/5807), [@antoninbas])
+- Upgrade CNI plugins to v1.4.0. ([#5747](https://github.com/antrea-io/antrea/pull/5747) [#5813](https://github.com/antrea-io/antrea/pull/5813), [@antoninbas] [@luolanzone])
+- Update the document for Egress feature's options and usage on AWS cloud. ([#5436](https://github.com/antrea-io/antrea/pull/5436), [@tnqn])
+- Add instructions to install Antrea in `encap` mode in AKS. ([#5901](https://github.com/antrea-io/antrea/pull/5901), [@antoninbas])
+- Add Flexible IPAM design details in `antrea-ipam.md`. ([#5339](https://github.com/antrea-io/antrea/pull/5339), [@gran-vmv])
+
+### Fixed
+
+- Store NetworkPolicy in filesystem as fallback data source to let antre-agent fallback to use the files if it can't connect to antrea-controller on startup. ([#5739](https://github.com/antrea-io/antrea/pull/5739), [@tnqn])
+- Enable Pod network after realizing initial NetworkPolicies to avoid traffic from/to Pods bypassing NetworkPolicy when antrea-agent restarts. ([#5777](https://github.com/antrea-io/antrea/pull/5777), [@tnqn])
+- Fix Clean-AntreaNetwork.ps1 invocation in Prepare-AntreaAgent.ps1 for containerized OVS on Windows. ([#5859](https://github.com/antrea-io/antrea/pull/5859), [@antoninbas])
+- Add missing space to kubelet args in Prepare-Node.ps1 so that kubelet can start successfully on Windows. ([#5858](https://github.com/antrea-io/antrea/pull/5858), [@antoninbas])
+- Fix `antctl trace-packet` command failure which is caused by arguments missing issue. ([#5838](https://github.com/antrea-io/antrea/pull/5838), [@luolanzone])
+- Support Local ExternalTrafficPolicy for Services with ExternalIPs when Antrea proxyAll mode is enabled. ([#5795](https://github.com/antrea-io/antrea/pull/5795), [@tnqn])
+- Set `net.ipv4.conf.antrea-gw0.arp_announce` to 1 to fix an ARP request leak when a Node or hostNetwork Pod accesses a local Pod and AntreaIPAM is enabled. ([#5657](https://github.com/antrea-io/antrea/pull/5657), [@gran-vmv])
+- Skip enforcement of ingress NetworkPolicies rules for hairpinned Service traffic (Pod accessing itself via a Service). ([#5687](https://github.com/antrea-io/antrea/pull/5687) [#5705](https://github.com/antrea-io/antrea/pull/5705), [@GraysonWu])
+- Add host-local IPAM GC on startup to avoid potential IP leak issue after antrea-agent restart. ([#5660](https://github.com/antrea-io/antrea/pull/5660), [@antoninbas])
+- Fix antrea-ubi image build failure with ubi8 image. ([#5723](https://github.com/antrea-io/antrea/pull/5723), [@antoninbas])
```

Comment:
this doesn't match the changelog entry we already have for 1.14.1:
```
$ git grep 5723
CHANGELOG-1.14.md:- Fix the CrashLookBackOff issue when using the UBI-based image. ([#5723](https://github.com/antrea-io/antrea/pull/5723), [@antoninbas])
```

Reply:
Fixed.